lensless/unrolled_admm.py

```diff
@@ -36,6 +37,7 @@ def __init__(
         psi_gram=None,
         pad=False,
         norm="backward",
+        post_process=None,
```

Comment:
so you've done this for "UnrolledADMM" but is there a way to do it for "TrainableReconstructionAlgorithm" so that it's applicable to "UnrolledFISTA" but anything else that inherits from it?